Formula/npiet.rb

```diff
@@ -0,0 +1,48 @@
+class Npiet < Formula
+  desc "Interpreter for the esoteric, picture-based Piet programming language"
+  homepage "https://www.bertnase.de/npiet"
+  url "https://www.bertnase.de/npiet/npiet-1.3e.tar.gz"
+  sha256 "e819c766d12e1dfaf71561429486ed85b6ae4438da6e7ee06871ab5ce58231c5"
+
+  depends_on "gd"
+  depends_on "libpng"
+  depends_on "homebrew/versions/giflib5"
+
+  def install
+    inreplace "npiet-foogol.c", "<malloc.h>", "<malloc/malloc.h>"
+    inreplace "npiet-foogol.y", "<malloc.h>", "<malloc/malloc.h>"
+
+    system "./configure", "--prefix=#{prefix}"
+    # --mandir doesn't work here, despite --help claiming it does.
+    inreplace "Makefile", "man/man1", "share/man/man1"
```

Comment:
The `Makefile` is generated by the `configure` script. I'll report that the `--mandir` option doesn't seem to be working upstream, but it's not just a simple patch to the `Makefile`. It has to be patched after `./configure` is run.